traits/trait_types.py

```diff
@@ -4321,8 +4339,7 @@ def validate(self, object, name, value):
                         "In the future, datetime.datetime instances will no "
                         "longer be accepted by this trait type. To accept "
                         "datetimes and silence this warning, use "
-                        "Date(allow_datetime=True) or "
-                        "Union(Datetime(), Date())."
+                        "Date(allow_datetime=True) or Union(Datetime, Date)."
```

Comment:
i'm not sure why this change was made.

Reply:
It shouldn't have been; I resolved a merge conflict with master the wrong way around.

Reply:
Fixed.